### ISSUE TITLE
Fake libretube website

### ISSUE THREAD

**Issue by terminal-cs** [libre-tube/LibreTube#4409]
### Steps to reproduce

N/a

### Expected behavior

N/a

### Actual behavior

N/a

### LibreTube version

N/a

### Android version

N/a

### Other details

This site is the first result on duckduckgo, and is clearly not an official site. 
![Screenshot_20230808-211946](https://github.com/libre-tube/LibreTube/assets/76945439/c6a974b0-9eee-4861-a3e3-fe4906891820)
![Screenshot_20230808-212253](https://github.com/libre-tube/LibreTube/assets/76945439/ca00d512-d7fa-410a-a3bc-a98de43821a5)

The actual download links appear to link to this repo's releases, but the whole site feels extremely unofficial and sketchy, and I haven't seen anything written about it here.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by terminal-cs** [comment]
I also think it has ads. I haven't seen them because I use AdBlock but they mentioned ads in the privacy policy

**Comment by Bnyro** [comment]
There's also libretube.net, which is not affiliated with us either.
The only official website is https://libre-tube.github.io, and any official social media accounts are referenced in the GitHub Readme!

**Comment by Bnyro** [comment]
Not sure if we can do anything to take the fake websites down, help is appreciated on that!

**Comment by terminal-cs** [comment]
> Not sure if we can do anything to take the fake websites down, help is appreciated on that!

I'm sure there's something, although personally I have no experience with that unfortunately.

**Comment by M00NJ** [comment]
I also don't think there is much we can do here. Except maybe adding a section to the readme with a warning. Something like this: "LibreTube doesn't have a website. Downloading the app from anywhere else than GitHub, F-Droid or IzzyOnDroid is highly discouraged due to the possibility of malware contamination."

**Comment by akano12** [comment]
I don't think there's a problem as long as the downloads are being pulled directly from github releases (unlike the modified android copies distributed on google play) .

At least we know nothing malicious is being installed on the devices of the users.

And after all, it's the end users' responsibility to do their researches before installing anything on their devices, we can't spoon-feed everyone and tell them what's good and what's bad

**Comment by akano12** [comment]
> maybe you could also show a prompt of the only official download locations (e.g. f-droid, github) when the user first opens the app but I don't know if that is too invasive.

The code is open sourced mate, they can simply remove the prompt and publish the app 🤣🤣🤣🤣🤣.

**Comment by Bnyro** [comment]
But it would inform users of LibreTube that there are clones of the app internet and that they have to make sure to only use official sources. I think that's actually not a bad idea.

**Comment by Bnyro** [comment]
And sometimes, I'm not sure if the people that just steal the app without respecting the license, even have any idea how to code - otherwise they wouldn't need to use our code.

**Comment by terminal-cs** [comment]
> And sometimes, I'm not sure if the people that just steal the app without respecting the license, even have any idea how to code - otherwise they wouldn't need to use our code.

make it hard to remove some how, like burried deep so that you'd only know how to remove it if you actually knew how to program. Many wouldn't even bother after than - maybe like a base-64 encoded string and a call to open a popup hidden somewhere - or even multiple calls so that if one doesn't trigger, another will. That will give them a headache for sure.

**Comment by olsonperrensen** [comment]
How about having a sticky notification on first run? It could contain a disclaimer stating the only trustworthy sources where the APK may be downloaded.

**Comment by FireMasterK** [comment]
> make it hard to remove some how, like burried deep so that you'd only know how to remove it if you actually knew how to program. Many wouldn't even bother after than - maybe like a base-64 encoded string and a call to open a popup hidden somewhere - or even multiple calls so that if one doesn't trigger, another will. That will give them a headache for sure.

That may not be GPL friendly since that's a bit like obfuscated code.

**Comment by Bluscream** [comment]
I would suggest a warning modal when the user first starts the app

**Comment by Doktor515** [comment]
https://libretube.app/

**Comment by Doktor515** [comment]
https://libretube.app/contact-us/

**Comment by Bnyro** [comment]
@MasterKia We now have our own domain: https://libretube.dev (alias to https://www.libretube.dev).

Could you please unblock that specific domain and redirect all the requests to https://libretube.dev instead of https://libre-tube.github.io?

Thank you!

**Comment by Bnyro** [comment]
> Fixed: [uBlockOrigin/uAssets@7e6d812](https://github.com/uBlockOrigin/uAssets/commit/7e6d812b5f77e1af056978a1659d2de3e4e8ac30)
>
Thank you!

**Comment by Bnyro** [comment]
> very fast but how do you get a free domain?
>
It's not a free domain, it's paid with donations.

**Comment by Darin755** [comment]
I'm not sure if anyone has filed a complaint yet but I noticed that this domain is registered under namecheap. You should report the abuse here 

https://www.namecheap.com/support/knowledgebase/article.aspx/9196/5/how-and-where-can-i-file-abuse-complaints/

Keep in mind only the owner of this project can complain. If you wait too long you will be unlikely to be able to take down other fake sites in the future as it sets precedent every time you ignore fake sites. (Not a lawyer)

**Comment by Darin755** [comment]
I couldn't help notice that this website seems to be hosted by openprovider. You can see how to report abuse here. https://support.openprovider.eu/hc/en-us/articles/360001725008-Reporting-abuse

I would strongly urge the owner of this repo to report the domain as it seems to be used for malware

**Comment by Darin755** [comment]
I couldn't help notice that this website seems to be hosted by openprovider. You can see how to report abuse here. https://support.openprovider.eu/hc/en-us/articles/360001725008-Reporting-abuse

I would strongly urge the owner of this repo to report the domain as it seems to be used for malware
